packages/nimble-components/src/table-column/date-text/index.ts

```diff
@@ -22,6 +50,76 @@ declare global {
  * The table column for displaying dates/times as text.
  */
 export class TableColumnDateText extends TableColumnTextBase {
+    @attr
+    public format?: 'custom';
+
+    @attr({ attribute: 'custom-locale-matcher' })
+    public customLocaleMatcher?: 'best fit' | 'lookup';
+
+    @attr({ attribute: 'custom-weekday' })
+    public customWeekday?: 'long' | 'short' | 'narrow';
+
+    @attr({ attribute: 'custom-era' })
+    public customEra?: 'long' | 'short' | 'narrow';
+
+    @attr({ attribute: 'custom-year' })
+    public customYear?: 'numeric' | '2-digit';
+
+    @attr({ attribute: 'custom-month' })
+    public customMonth?: 'numeric' | '2-digit' | 'long' | 'short' | 'narrow';
+
+    @attr({ attribute: 'custom-day' })
+    public customDay?: 'numeric' | '2-digit';
+
+    @attr({ attribute: 'custom-hour' })
+    public customHour?: 'numeric' | '2-digit';
+
+    @attr({ attribute: 'custom-minute' })
+    public customMinute?: 'numeric' | '2-digit';
+
+    @attr({ attribute: 'custom-second' })
+    public customSecond?: 'numeric' | '2-digit';
+
+    @attr({ attribute: 'custom-time-zone-name' })
+    public customTimeZoneName?:
+    | 'short'
+    | 'long'
+    | 'shortOffset'
+    | 'longOffset'
+    | 'shortGeneric'
+    | 'longGeneric';
+
+    @attr({ attribute: 'custom-format-matcher' })
+    public customFormatMatcher?: 'best fit' | 'basic';
+
+    @attr({ attribute: 'custom-hour12', mode: 'boolean' })
+    public customHour12?: boolean;
+
+    @attr({ attribute: 'custom-time-zone' })
+    public customTimeZone?: string;
+
+    @attr({ attribute: 'custom-calendar' })
+    public customCalendar?: string;
+
+    @attr({ attribute: 'custom-day-period' })
+    public customDayPeriod?: 'narrow' | 'short' | 'long';
+
+    @attr({ attribute: 'custom-numbering-system' })
+    public customNumberingSystem?: string;
+
+    @attr({ attribute: 'custom-date-style' })
+    public customDateStyle?: 'full' | 'long' | 'medium' | 'short';
+
+    @attr({ attribute: 'custom-time-style' })
+    public customTimeStyle?: 'full' | 'long' | 'medium' | 'short';
+
+    @attr({ attribute: 'custom-hour-cycle' })
+    public customHourCycle?: 'h11' | 'h12' | 'h23' | 'h24';
+
+    public override connectedCallback(): void {
+        this.updateColumnConfig();
```

Comment:
I could be wrong, but I was thinking that you didn't need to call your change handler within the `connectedCallback` if none of the `xyzChanged()` functions returned early when the component isn't connected. Is that not the case? Did you find that overriding `connectedCallback` was required? If it is required, you need to make sure to call `super.connectedCallback()`.

Reply:
Without it, the group headers don't render a date. Somehow the TableColumnDateTextGroupHeaderView doesn't get a columnConfig.